### DISCUSSION TITLE
php session not crypted

### DISCUSSION THREAD

**Discussion by ffPjrZUGXfcxuAj** [Studio-42/elFinder#3329] [Q&A]
in php session i have 
`"user";s:10:"USER";s:4:"pass";s:13:"PASS";`
USER and PASS are not crypted and all session is not crypeted

**Comment by nao-pon** [comment]
@camaran You can use your own session handler.

- https://github.com/Studio-42/elFinder/wiki/Connector-configuration-options-2.1#session-218
